### ISSUE TITLE
controlbar pushed away from headerbar if titlebar is too long.

### ISSUE THREAD

**Issue by Bryuk** [Flyer53/jsPanel3#69]
I have issue with panel controls that gets pushed if title is too long. My title is dynamic and can be longer sometime.
Can you help making controls to always stay in right corner and when resize only title gets smaller, but controls always stay the same.

![image](https://user-images.githubusercontent.com/6089124/45038293-f3cbbd00-b02e-11e8-82bd-bbbc5d376e57.png)

**Comment by Flyer53** [comment]
@Bryuk 
I guess you use IE11 or 10?
Please try to add the following CSS to your CSS after loading the jsPanel CSS file
```css
.jsPanel-titlebar{
    flex: 1 1 0px  // jsPanel css file uses flex: 1 1 auto
}
```
or change the jsPanel css file accordingly. IE obviously can't handle `flex-basis: auto` properly.

Please let me know how it works ...

**Comment by Bryuk** [comment]
Thanks! This solved my issue.
I have to learn flex css better...

**Comment by Flyer53** [comment]
Not your fault. I was not aware of this glitch either ... so thanks for reporting.

**Comment by Flyer53** [comment]
Fixed with v3.11.2
